lib/api/api.js

```diff
@@ -190,7 +190,13 @@ const api = {
         let fileEventData = null;
 
         if (apiMethod === 'objectPost') {
-            formDataParser = busboy({ headers: request.headers });
+            if (request.headers['content-type'].includes('multipart/form-data')) {
```

Comment:
This crash if `content-type` is not defined
```
TypeError: Cannot read properties of undefined (reading 'includes')
```

Also it still crash if `Content-Type: a multipart/form-data`

Also with invalid boundary aws show different message than busboy does